test/browser.ts

```diff
@@ -19,8 +19,8 @@ describe('libp2p-webtransport', () => {
     const maStr: string = process.env.serverAddr!
     const ma = multiaddr(maStr)
     const node = await createLibp2p({
-      transports: [new WebTransportLibp2p()],
-      connectionEncryption: [new Noise()]
+      transports: [webTransport()],
+      connectionEncryption: [() => new Noise()]
```

Comment:
This is needed until https://github.com/ChainSafe/js-libp2p-noise/pull/229 is merged